superset/databases/schemas.py

```diff
@@ -429,3 +433,12 @@ class ImportV1DatabaseSchema(Schema):
     extra = fields.Nested(ImportV1DatabaseExtraSchema)
     uuid = fields.UUID(required=True)
     version = fields.String(required=True)
+
+    # pylint: disable=no-self-use, unused-argument
+    @validates_schema
+    def validate_password(self, data: Dict[str, Any], **kwargs: Any) -> None:
+        """If sqlalchemy_uri has a masked password, password is required"""
+        uri = data["sqlalchemy_uri"]
```

Comment:
will we always receive `sqlalchemy_uri` key from data?

Reply:
Yes, if `sqlalchemy_uri` is not there the validator will raise an exception before calling this method, since we have:

```python
sqlalchemy_uri = fields.String(required=True)
```